### DISCUSSION TITLE
Is it possible to play multiple audio files at the same time ?

### DISCUSSION THREAD

**Discussion by armandchbr** [dooboolab/react-native-audio-recorder-player#375] [Q&A]
Hi, I'm developing an app where I need to record multiple audio file, and I'll probably need to listen to them at the same time. Everything work fine with this lib, but when I try to play another file, I have to wait for the first one to be finished otherwise it won't play. Is there anything I can do ?

Thank you

**Comment by hyochan** [comment]
Currently no. It is not designed to support multiple audio.
